scripts/provision-namespace.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+
+execute_kubectl() {
+  local command=$1
+  if $DRY_RUN; then
+    echo "kubectl --dry-run=client $command "
+    kubectl --dry-run=client $command
+  else
+    echo "kubectl $command"
+    kubectl $command
+  fi
+}
+
+execute_oc() {
+  local command=$1
+  if $DRY_RUN; then
+    echo "oc $command "
+  else
+    echo "oc $command"
+    oc $command
+  fi
+}
+
+# Set default values
+DRY_RUN=false
+
+# Display usage
+function usage() {
+  echo "Usage: $0 [options]"
+  echo "Options:"
+  echo "  -n, --namespace     <namespace>                        The namespace on the QShift cluster (mandatory)"
+  echo "  -q, --quay          <quay_username:quay_password>      The Quay username and password (mandatory)"
+  echo "  -o, --quay-org      <quay_username:quay_password>      The Quay organization hosting the images (mandatory)"
```

Comment:
This is confusing. What do I need to add here? org, username ? password ?

Reply:
The quay.io registry allows to push images under different organizations and creds can be defined at the root of quay.io or quay.io/ORG => 
```
{
  "auths": {
    "quay.io/$QUAY_ORG": {
      "auth": "$QUAY_CREDS_BASE64"
    },
```
By doing that you have the guarantee that the token/creds matches the ORG level where such a robot token has been defined

`Robot Accounts are named tokens that can be granted permissions on multiple repositories under this organization. They are typically used in environments where credentials will be shared, such as deployment systems. `

Reply:
I reviewed the wording. See commit : f6298c